libsolidity/lsp/LanguageServer.cpp

```diff
@@ -101,14 +101,29 @@ Json::Value LanguageServer::toJson(SourceLocation const& _location)
 void LanguageServer::changeConfiguration(Json::Value const& _settings)
 {
 	m_settingsObject = _settings;
+	Json::Value jsonIncludePaths = _settings["include-paths"];
+	if (jsonIncludePaths && jsonIncludePaths.isArray())
+	{
+		std::vector<boost::filesystem::path> includePaths;
+		for (Json::Value jsonPath: jsonIncludePaths)
+		{
+			if (!jsonPath.isString())
+			{
+				m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
+				continue;
+			}
+			includePaths.emplace_back(boost::filesystem::path(jsonPath.asString()));
```

Comment:
Isn't this equivalent to a simple `if`/`else`?
```suggestion
			if (!jsonPath.isString())
				m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
			else
				includePaths.emplace_back(boost::filesystem::path(jsonPath.asString()));
```

Reply:
I wasn't sure about what style to chose. But yes ;)